### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by HITGIF**
- Fix up the structure that was all messed up
- Clean up redundant parts
- Add gitter chat badge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,92 +49,101 @@ $ git clone https://github.com/jboss-outreach/facebook-echo-bot
 #### For Windows
 
  1. First you need to download and install [Git](https://git-scm.com/downloads) (if it is not installed).
- 2. Download it from here(https://git-scm.com/download/win) 
+ 2. Download it from here(https://git-scm.com/download/win)
  3. Run the installer
  4. Download and install maven from here(https://maven.apache.org/download.cgi)
  5. After all this, you need to clone this repository which you can do it with the given code or download[.zip](https://github.com/jboss-outreach/facebook-echo-bot/archive/master.zip) file:
```
Comment (aashutoshrathi) [inline README.md]: Give space after download and give file name as `zip file`.

**Comment by HITGIF** [comment]
Done :) @aashutoshrathi

**Comment by bhavyaagg** [comment]
@HITGIF Please resolve the conflicts. Another PR which was created before this has been merged.

**Comment by HITGIF** [comment]
Conflicts resolved :)

**Comment by aashutoshrathi** [comment]
👍

**Pull request merged**